packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -57,8 +58,13 @@ export class Gauge extends PureComponent<Props> {
     const thresholds = field.thresholds ?? Gauge.defaultProps.field?.thresholds!;
     const isPercent = thresholds.mode === ThresholdsMode.Percentage;
     const steps = thresholds.steps;
-    let min = field.min!;
-    let max = field.max!;
+    let min = field.min;
+    let max = field.max;
+
+    if (!isNumber(min) || !isNumber(max)) {
```

Comment:
@torkelo do you know if we can end up in a scenario where we have a value for `min` but not `max` and the other way around? Thinking if I should change this one to `&&` and use a default value for `min` or `max` in that scenario.

Reply:
yes, I think technically it is possible. wonder if apply field overrides always should set some numeric defaults for min & max so that logic like this isn't needed in every visualization. 

thoughts @ryantxu ?

Reply:
@torkelo maybe the autoMinMax should be true by default in the `ApplyFieldOverrideOptions`? then min/max would be guaranteed.

Reply:
yes, it is always true now in PanelQueryRunner

Reply:
OK ok, so what do you think I should default to in case one of them is a numeric value but not the other?

Reply:
Think the fix should not be here but in maybe here? https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/field/fieldDisplay.ts#L260 

The bug is that min & max are not numeric when there is no data right? think that the no data field config options are defined there

Reply:
ah, I will have a look at it. Thanks for the feedback!